src/resources/js/tribe-events.js

```diff
@@ -1408,6 +1408,15 @@ Date.prototype.format = function( mask, utc ) {
 					}
 				}
 			}
+
+			if ( ( ts.view === 'month' ) && ( ! $( '#tribe-events-bar' ).length ) ) {
```

Comment:
Yoda first conditional, and remove extra parenthesis on each one of the conditionals